cmd/policy-assistant/pkg/matcher/traffic.go

```diff
@@ -250,16 +253,22 @@ func ReplicaSetsToTrafficPeers() []TrafficPeer {
 		if err != nil {
 			logrus.Fatalf("unable to read replicaSets from kube, ns '%s': %+v", namespace.Name, err)
 		}
+
 		for _, replicaSet := range kubeReplicaSets {
-			TmpInternalPeer := InternalPeer{
-				Workload: namespace.Name + "/replicaset/" + replicaSet.Name,
-			}
-			TmpPeer := TrafficPeer{
-				Internal: &TmpInternalPeer,
-			}
-			TmpPeerTranslated := TmpPeer.Translate()
-			if TmpPeerTranslated.Internal.Workload != "" {
-				replicaSetPeers = append(replicaSetPeers, TmpPeerTranslated)
+			if replicaSet.OwnerReferences != nil {
+				logrus.Infof("replicaset already handled")
```

Comment:
nit: this could be a noisy log
```suggestion
				continue
```

Reply:
same thing in `PodsToTrafficPeers()`

Reply:
done